packages/toolkit/project/gitlab/.gitlab-ci.tmpl

```diff
@@ -0,0 +1,47 @@
+# This file was autogenerated by 10up Toolkit.
+
+image: docker.10up.com/10up-build/deploy-container:latest
+
+include:
+  - project: 10up-build/ci-library
+    ref: trunk
+    file: /main.yml
+
+setup_10up_toolkit:
```

Comment:
Instead of cloning 10up-toolking and installing it from scratch why not use `npx 10up-toolkit project init`. `npx` will use by default the latest version in npm.

Reply:
Of course we'd need to release it first to be able to test this.

Reply:
100%. This will be removed once we release. Chicken and egg problem